source/agora/network/NetworkManager.d

```diff
@@ -502,12 +502,11 @@ public class NetworkManager
     }
 
     /// Returns: the list of node IPs this node is connected to
-    public NodeInfo getNetworkInfo () pure nothrow @safe @nogc
+    public void fillNetworkInfo (ref NetworkState state, ref Set!string addresses)
+        pure nothrow @safe @nogc
     {
-        return NodeInfo(
-            this.minPeersConnected()
-                ? NetworkState.Complete : NetworkState.Incomplete,
-            this.known_addresses);
+        state = this.minPeersConnected() ? NetworkState.Complete : NetworkState.Incomplete;
+        addresses = this.known_addresses;
```

Comment:
Looks unrelated (bad rebase?)

Reply:
This is the first commit.
I excluded it because it is not relevant.